gcovr/utils.py

```diff
@@ -26,7 +26,7 @@
 import sys
 from contextlib import contextmanager
 
-from .coverage import CovData, CoverageStat, SEPARATOR
+from .coverage import CovData, CoverageStat, GCOVR_PATH_SEPARATOR
```

Comment:
The dependency from utils to coverage must be removed to be able to import the `force_unix_separator` in coverage.py. I think the function `sort_coverage` is better implemented in coverage.py than in utils.py.

```suggestion
GCOVR_PATH_SEPARATOR = "/"
```

Reply:
But it is tied to the coverage classes defined in coverage.py. It could be a member function of `CovData`.

Reply:
I moved the function to `coverage.py`.